src/renderer/components/common/QrCode/common/types.ts

```diff
@@ -37,6 +38,10 @@ export type SeedInfo = {
   derivedKeys: AddressInfo[];
 };
 
+export type SeedInfoExtended = SeedInfo & {
```

Comment:
Seems strange that it's an extended type but the property is optional
Can we make it required? If not, can we put it into existing `SeedInfo` and make optional?